api/contrib/envoy/extensions/config/v3alpha/kv_store_xds_delegate_config.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.extensions.config.v3alpha;
+
+import "envoy/config/common/key_value/v3/config.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.config.v3alpha";
+option java_outer_classname = "KvStoreXdsDelegateConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/config/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#extension: envoy.xds_delegates.kv_store]
+//
+// Configuration for a KeyValueStore-based XdsResourcesDelegate implementation. This implementation
+// updates the underlying KV store with xDS resources received from the configured management
+// servers, enabling configuration to be persisted locally and used on startup in case connectivity
+// with the xDS management servers could not be established.
+//
+// [#not-implemented-hide:]
```

Comment:
I don't think we need to hide this given it's in contrib

Reply:
Removed